qemu/tests/cfg/throttle_utils_test.cfg

```diff
@@ -0,0 +1,36 @@
+- throttle_utils_test:
+    only Linux
+    type = throttle_utils_test
+    sleep_time = 5
```

Comment:
add qemu_force_use_drive_expression = no? to explicitly tell that the case must run in blockdev mode